changelog/1373.doc.rst

```diff
@@ -1,2 +1,2 @@
 Described additional environments for building the documentation with make_
-in the documentation guide.
+in the |documentation guide|.
```

Comment:
Not rendering correctly on RTD.

![image](https://user-images.githubusercontent.com/29869348/155810594-d9c1b4b1-ed16-4cc8-a3f8-fd17b999b1d7.png)